app/containers/Tracks/index.js

```diff
@@ -100,6 +102,26 @@ export function Track({
     }
   };
 
+  const handlePlay = (item) => {
+    if (item.previewUrl !== trackurl) {
+      setTrackurl(item.previewUrl);
+      setIsplaying(true);
+    } else {
+      setIsplaying(false);
+    }
+  };
+  useEffect(() => {
+    const audioEl = document.getElementsByClassName('audio-element')[0];
```

Comment:
Just check once if it is a good practise to use useEffect without a dependency array. Because if I am not wrong this useEffect will always get called when any type of state changes. Instead can we do 2 things -
 first lets use useRef and assign the audio tag reference to that variable
 Second using the useRef variable now whenever the handlePlay function is called lets do the respective audio play/pause functionality